test/transformationsTest.js

```diff
@@ -0,0 +1,291 @@
+'use strict'
+
+const test = require('tape')
+const transforms = require('../src/transform/handler')
+
+test.test('Transformations', t => {
+  t.test('dateTimeToDate()', t => {
+    t.test('should convert iso datetime string to date', t => {
+      const inputDate = '1988-03-24T00:00:00.000+0800'
+      const expected = '1988-03-24'
+
+      const result = transforms['dateTimeToDate'](inputDate)
+
+      t.equal(result, expected)
+      t.end()
+    })
+  })
+
+  t.test('getGenderFromCode()', t => {
+    t.test('should convert single letter code to word', t => {
+      const inputCode = 'F'
+      const expectedOutput = 'female'
+
+      const functionParameters = {
+        F: 'female',
+        M: 'male',
+        O: 'other'
+      }
+
+      const result = transforms['mapCodes'](
+        inputCode,
+        null,
+        null,
+        null,
+        null,
+        functionParameters
+      )
+
+      t.equal(result, expectedOutput)
+      t.end()
+    })
+
+    t.test('should convert unhandled code to null', t => {
```

Comment:
Im not sure this test tile reflects the actual test cse

Reply:
:| Whoops